app/src/main/java/com/example/polina/socialnetwork/ServerAPI.java

```diff
@@ -137,5 +132,42 @@ private JSONObject logInSignUp (String email, String password, String path, Cont
         }
     }
 
+    private JSONObject postRequest(Context context, JSONObject o, String path) {
+        try {
+            DefaultHttpClient client = new DefaultHttpClient();
+            HttpPost post = new HttpPost(path);
+            CookieSyncManager.createInstance(context);
+            CookieManager cookieManager = CookieManager.getInstance();
+            post.addHeader("Cookie", cookieManager.getCookie(path));
+            String data = o.toString();
+
+            post.setEntity(new StringEntity(data, HTTP.UTF_8));
+            HttpResponse resp = client.execute(post);
+
+            if (resp.getStatusLine().getStatusCode() < 400) {
+                String s = EntityUtils.toString(resp.getEntity());
+                return new JSONObject(s);
+            } else return null;
+        } catch (Exception e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    public JSONObject newPost(Context context, String massage, String location, String attachment, String image, String account) {
```

Comment:
please use Location class (not String). location should be stored like:
"location" : {"latitude": 12.543534, "longitude": 32,3213123}